src/components/TagActions/TagActions.tsx

```
@@ -353,17 +348,36 @@
+                          <TagNode
+                            tag={tag}
+                            checkedTags={checkedTags}
+                            toggleTagSelection={toggleTagSelection}
+                            t={t}
+                          />
+                        </div>
+
+                        {/* Ancestor tags breadcrumbs positioned at the end of TagNode */}
+                        {tag.parentTag && (
+                          <div className="position-absolute end-0 top-0 d-flex flex-row mt-2 me-3 pt-0 text-secondary">
+                            <>{'('}</>
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)